source/common/grpc/context_impl.cc

```diff
@@ -114,6 +114,21 @@ void ContextImpl::chargeResponseMessageStat(const Upstream::ClusterInfo& cluster
       .add(amount);
 }
 
+void ContextImpl::chargeUpstreamStat(const Upstream::ClusterInfo& cluster,
+                                     const absl::optional<RequestStatNames>& request_names,
+                                     uint64_t duration) {
```

Comment:
change type to `std::chrono::duration`

Reply:
Would `miliseconds` work? duration is a template, pretty sure I'll need to modify the class as well :thinking: